tensorflow/lite/allocation.h

```diff
@@ -24,6 +24,9 @@ limitations under the License.
 #include "tensorflow/lite/core/api/error_reporter.h"
 #include "tensorflow/lite/simple_memory_arena.h"
 #include "tensorflow/lite/string.h"
+#ifdef _WIN32
+#include <windows.h>
```

Comment:
We shouldn't need these changes if we switch to using mmap_allocation_disabled.cc. See comment below.